.github/workflows/ci.yml

```diff
@@ -164,7 +164,7 @@ jobs:
         shell: bash
         env:
           TESTPAINT: true
-        run: . scripts/setenv -q && build -DWITH_TESTS=on -DCMAKE_BUILD_TYPE=MinSizeRel -DBUILD_SHARED_LIBS=ON -DPORTABLE=ON -DFORCE32=ON -DCMAKE_CXX_FLAGS="-m32 -gz"
+        run: . scripts/setenv -q && build -DWITH_TESTS=on -DCMAKE_BUILD_TYPE=MinSizeRel -DBUILD_SHARED_LIBS=ON -DPORTABLE=ON -DFORCE32=ON -DENABLE_SCRIPTING=OFF -DCMAKE_CXX_FLAGS="-m32 -gz"
```

Comment:
Is this correct? Seems like we didn't need to disable scripting previously

Reply:
No, but that was because we didn't include any file that includes references to scripting. So I would say disabling scripting for TestPaint is correct, it just happened to work by accident previously.

Reply:
Hmm, I see. Do we intend for it to work in the future? Because if so, we should open an issue to track this limitation

Reply:
I can't think of a reason why scripting support would be useful for TestPaint. It would be like giving a bicycle to a fish.

Reply:
Also testpaint in its current form needs to die and be reborn completely different.